integration-tests/delete-partition.py

```diff
@@ -22,5 +22,6 @@
 
 print staging
 
+storage.calculate_actiongraph()
```

Comment:
commit could calculate the actiongraph if it was never calculated before. But it does not know when to recalculate it after it was invalidated. That is the task of the user (see Storage.h).